src/cid.ts

```diff
@@ -3,62 +3,39 @@ import * as Digest from './hashes/digest.js'
 import { base58btc } from './bases/base58.js'
 import { base32 } from './bases/base32.js'
 import { coerce } from './bytes.js'
-// Linter can see that API is used in types.
-// eslint-disable-next-line
-import * as API from "./link/interface.js"
+import type * as API from './link/interface.js'
```

Comment:
we're still importing these `interface.js` files even though they got deleted, does that make sense?

Reply:
It's referring to the types in `./link/interface.ts`.
All imports use the extension of the output (".js") rather than ".ts".
In this case it does seem a little strange, considering that type imports are stripped from the generated javascript, but thats just how its done.